docs/api-reference/web-mercator-utils.md

```diff
@@ -201,3 +201,23 @@ Parameters:
 Returns:
 - `{longitude, latitude, zoom}`
 
+### `getFlyToDuration(startProps, endProps, opts)`
+
+Returns time in milliseconds, that is required perform transition from one viewport to another. Time returned is proportional to the distance we are transitioning to. This util function implements the algorithm described in “Smooth and efficient zooming and panning.” by Jarke J. van Wijk and Wim A.A. Nuij.
+
+Parameters:
+- `startProps` (Object) - viewport to fly from
+- `startProps.width` (Number, required)
```

Comment:
Use nested list?

Reply:
Rest of the documents have the same style.